pkg/controller/containerjfr/resource_definitions/resource_definitions.go

```diff
@@ -420,6 +419,12 @@ func NewGrafanaContainer(cr *rhjmcv1alpha1.ContainerJFR, tls *TLSConfig) corev1.
 	}
 }
 
+const datasourceHost = "127.0.0.1"
+const datasourcePort = "8080"
+
+// DatasourceURL contains the fixed URL to jfr-datasource's web server
+const DatasourceURL = "http://" + datasourceHost + datasourcePort
```

Comment:
I think this is missing a `:` to separate the host and port, isn't it?

Reply:
So there is. Thanks for catching it. Fixed now!